<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReloadDialog</class>
 <widget class="QDialog" name="ReloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File was changed</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioReload">
       <property name="text">
        <string>Don't ask and reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioIgnore">
       <property name="text">
        <string>Don't ask and ignore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAsk">
       <property name="text">
        <string>Always ask</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reloadBtn">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancleBtn">
       <property name="text">
        <string>Ignore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>reloadBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReloadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancleBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReloadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
